tests/test_suites/ShortSystemTests/test_chunkserver_masterconn_workers.sh

```diff
@@ -0,0 +1,68 @@
+CHUNKSERVERS=3 \
+	USE_RAMDISK=YES \
+	MASTER_CUSTOM_GOALS="1 ec21: \$ec(2,1)" \
+	AUTO_SHADOW_MASTER="NO" \
+	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER" \
+	CHUNKSERVER_EXTRA_CONFIG="MAGIC_DEBUG_LOG = $TEMP_DIR/log|LOG_FLUSH_ON=INFO" \
+	setup_local_empty_saunafs info
+
+cd "${info[mount0]}"
+
+# Generate and validate some files in parallel
+function generateAndValidateFiles() {
+	for file in $(seq 1 20); do
+		size=$((file * 1024 * 1024))
+		FILE_SIZE=${size} assert_success file-generate "file.${file}" &
+	done
+
+	wait
+
+	for file in $(seq 1 20); do
+		size=$((file * 1024 * 1024))
+		FILE_SIZE=${size} assert_success file-validate "file.${file}" &
```

Comment:
```suggestion
		assert_success file-validate "file.${file}" &
```

Reply:
FILE_SIZE is not required for validating it...

Reply:
Done, thanks.